code/modules/mining/abandonedcrates.dm

```diff
@@ -118,15 +118,6 @@
 			new /obj/item/material/sword/katana(src)
 		if(93)
 			new /obj/item/dnainjector/xraymut(src) // Probably the least OP
-		if(94) // Why the hell not
-			new /obj/item/storage/backpack/clown(src)
-			new /obj/item/clothing/under/rank/clown(src)
-			new /obj/item/clothing/shoes/clown_shoes(src)
-			new /obj/item/modular_computer/handheld/pda/civilian/clown(src)
-			new /obj/item/clothing/mask/gas/clown_hat(src)
-			new /obj/item/bikehorn(src)
-			new /obj/item/pen/crayon/rainbow(src)
-			new /obj/item/reagent_containers/spray/waterflower(src)
```

Comment:
This needs something to replace it as loot.

Reply:
I hate Clowns